src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
why do you need a hash?

it's possible to configure multiple plugins, but each should have an unique config path